packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -1,40 +1,131 @@
+import { FocusTrap } from '@strapi/parts/FocusTrap';
 import { useConfigurations } from '../../../hooks';
-import StaticLinks from './StaticLinks';
-import Wrapper from './Wrapper';
+
+const Wrapper = styled.button`
+  position: fixed;
+  bottom: ${({ theme }) => theme.spaces[2]};
+  right: ${({ theme }) => theme.spaces[2]};
+  width: ${({ theme }) => theme.spaces[8]};
+  height: ${({ theme }) => theme.spaces[8]};
+  background: ${({ theme }) => theme.colors.primary600};
+  box-shadow: ${({ theme }) => theme.shadows.tableShadow};
+  border-radius: 50%;
+
+  svg {
+    color: white;
+  }
+`;
+
+const LinksWrapper = styled(Box)`
+  position: absolute;
+  bottom: ${({ theme }) => `${theme.spaces[9]}`};
+  right: 0;
+  width: ${200 / 16}rem;
+`;
+
+const StyledLink = styled.a`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  text-decoration: none;
+  padding: ${({ theme }) => theme.spaces[2]};
+  padding-left: ${({ theme }) => theme.spaces[5]};
+
+  svg {
+    color: ${({ theme }) => theme.colors.neutral600};
+    margin-right: ${({ theme }) => theme.spaces[2]};
+  }
+
+  &:hover {
+    background: ${({ theme }) => theme.colors.neutral100};
+    color: ${({ theme }) => theme.colors.neutral500};
+
+    svg {
+      color: ${({ theme }) => theme.colors.neutral700};
+    }
+
+    ${[Text]} {
+      color: ${({ theme }) => theme.colors.neutral700};
+    }
+  }
+
+  ${[Text]} {
+    color: ${({ theme }) => theme.colors.neutral600};
+  }
+`;
 
 const Onboarding = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const { formatMessage } = useIntl();
   const { showTutorials } = useConfigurations();
 
   if (!showTutorials) {
     return null;
   }
 
-  return <OnboardingVideos />;
-};
-
-const OnboardingVideos = () => {
-  const [isOpen, setIsOpen] = useState(false);
+  const staticLinks = [
+    {
+      icon: 'book',
+      label: formatMessage({
+        id: 'app.components.LeftMenuFooter.documentation',
+        defaultMessage: 'Documentation',
+      }),
+      destination: 'https://strapi.io/documentation',
+    },
+    {
+      icon: 'file',
+      label: formatMessage({ id: 'app.static.links.cheatsheet', defaultMessage: 'CheatSheet' }),
+      destination: 'https://strapi-showcase.s3-us-west-2.amazonaws.com/CheatSheet.pdf',
+    },
+  ];
 
   const handleClick = () => {
     setIsOpen(prev => !prev);
   };
 
   return (
-    <Wrapper className="visible" isOpen={isOpen}>
-      <div className={cn('videosContent', isOpen ? 'shown' : 'hide')}>
-        <StaticLinks />
-      </div>
-      <div className="openBtn">
-        <button onClick={handleClick} className={isOpen ? 'active' : ''} type="button">
-          <FontAwesomeIcon icon={faQuestion} />
-          <FontAwesomeIcon icon={faTimes} />
-          <span />
-        </button>
-      </div>
+    <Wrapper
+      aria-label={formatMessage({
+        id: 'app.components.Onboarding.help.button',
+        defaultMessage: 'Help Button',
+      })}
+      type="button"
```

Comment:
Adding the type is only needed when the button is in a <form> tag to prevent its submission

Reply:
You're right but maybe we should normalize having the type="button" always there. EG: in Parts we needed to put type button for the accordion because it was triggering forms.